### ISSUE TITLE
New Variant Proposal: Starter

### ISSUE THREAD

**Issue by iamwhoiamhahaha** [Zamiell/hanabi-live#2217]
### Proposal Origin
- I got this idea when kakashi mentioned pokemon in the dialog (I was asking for his idea about the new names for the variants I proposed [here](https://github.com/Zamiell/hanabi-live/issues/2215))
> IAMJEFF:
> like reboot?
> Number Reboot
> 
> Kakashi:
> lol yea that sounds a lot better than our other ideas for a name
> is that the best we can do?
> it's like taking our clues to the pokemon healer
> can we be creative somehow? that's why I mentioned the pokemon healing
- In Pokemon Red/Blue, the player has to choose one of the Starter Pokemon (says Bulbusaur, Charmander, and Squirtle) to start their journey. Once they pick one, they are not able to get the other two Starter Pokemon for the rest of the game unless they trade Pokemon from other players.
- In a Hanabi *3 Suits* game, there are three types of color clues you are allowed to give, i.e., Red clue, Green clue, and Blue clue, which represents Charmander, Bulbasuar, and Squirtle, respectively.
- In conclusion, based on the idea of "playing hanabi like a pokemon game", the players are not allowed to give the other types of color clues once they choose a specific one, e.g., a red clue (as a Charmander).

#### Color Starter (N Suits)
- Only `N-2` types of color clues can be given during the game.

#### Number Starter (or Number M Starter)
- Only `M` types of number clues can be given during the game.
#### All Starter
- Only `M` types of number clues can be given during the game.
- Only `N-2` types of color clues can be given during the game.

**Comment by Zamiell** [comment]
there are already variants in which you only get to clue one color and that color does not get to touch all cards, e.g. extremely ambiguous & white

in a variant where you only get to clue one color and that color does not touch most of the cards, that would play very similarly to color mute, albeit slightly easier

thus, it doesn't strike me as being particularly unique or very fun
